tests/nightly/test_large_vector.py

```diff
@@ -708,6 +708,182 @@ def test_full():
     assert a[-1] == 3
 
 
+def test_astype():
+    x = create_vector(size=LARGE_X//4)
+    x = nd.tile(x, 4)
+    y = x.astype('int32')
+    assert y.dtype == np.int32
+    assert y[-1] == LARGE_X//4-1
+
+
+def test_cast():
+    x = create_vector(size=LARGE_X//4)
+    x = nd.tile(x, 4)
+    y = nd.cast(x, np.int32)
+    assert y.dtype == np.int32
+    assert y[-1] == LARGE_X//4-1
+
+
+def test_repeat():
+    x = create_vector(size=LARGE_X//2)
+    y = nd.repeat(x, repeats=2, axis = 0)
+    assert y.shape[0] == LARGE_X
+    assert y[1] == 0
+    assert y[LARGE_X-1] == LARGE_X//2-1
+
+
+def create_input_for_rounding_ops():
+    inp = nd.arange(-LARGE_X//2, LARGE_X//2, dtype=np.float64)
+    inp = inp/2
+    return inp
+
+
+def test_ceil():
+    x = create_input_for_rounding_ops()
+    y = nd.ceil(x)
+    assert y[LARGE_X//2-2] == -1
+    assert y[LARGE_X//2-1] == 0
+    assert y[LARGE_X//2] == 0
+    assert y[LARGE_X//2+1] == 1
+    assert y[LARGE_X//2+2] == 1
+
+
+def test_fix():
+    x = create_input_for_rounding_ops()
+    y = nd.fix(x)
+    assert y[LARGE_X//2-2] == -1
```

Comment:
can you add some comment of what it is testing, here and for other tests.

Reply:
aren't docs supposed to explain what a particular function is supposed to be doing? http://mxnet.incubator.apache.org/api/python/docs/api/ndarray/ndarray.html#mxnet.ndarray.fix

Reply:
my request is to explain the tests: why only these specific positions are checked as -1 or 0 , something to say x was this before we applied fix on it and what we get is expected because of this.

Reply:
ok got it. I will add comments explaining this. Good point